docs/en/administrator-guide/config/be_config.md

```diff
@@ -678,6 +678,18 @@ Default：10737418240
 
 BloomFilter/Min/Max and other statistical information cache capacity
 
+### `kafka_api_version_request`
```

Comment:
Could you please update Routine Load doc ? https://doris.apache.org/master/zh-CN/administrator-guide/load-data/routine-load-manual.html#%E5%8E%9F%E7%90%86

Reply:
done